### ISSUE TITLE
[3.5.3] Testing

### ISSUE THREAD

**Issue by KeeHanXiang** [lowyisan/INF2001_P6-6#139]
**What is the task about?** 
This task is focused on the documentation of the system's Testing. 

**What are the goals of the task?** 
Successfully document the Black Box testing process conducted on the implemented design. 

**Success or Completion Criteria:** 
Thorough documentation of Black Box and White Box testing procedures, including test execution details.

**Allocated time:** 4 days

**Start date and end date:** [Start Date: 16/11/23], [End Date: 19/11/23]

**Owner:** Han Xiang & Yi San

**Status of the task:** To-Do

**Priority:** High

**Comment by KeeHanXiang** [comment]
Describe the work done in detail.
This task pertains to documenting the system's Testing processes.

Has this work been committed to the repository?
The work does not involve any code. Therefore, no repository commits is needed.

How does the work affect success or completion criteria?
The success or completion criteria for this task include the comprehensive documentation of both Black Box and White Box testing procedures, encompassing detailed test execution information.

Start date and end date:
[Start Date: 16/11/23]
[End Date: 19/11/23]

**Comment by lowyisan** [comment]
**Describe the work done in detail.**
I worked on documenting the White Box testing process we conducted on our software system. This entailed a detailed recording of our internal code testing, including the logic paths and code coverage. The login method was meticulously documented, covering the inputs, the code behavior examined, and the outcomes.

**Has this work been committed to the repository?**
The work does not involve any code. Therefore, no repository commits is needed.

**How does the work affect success or completion criteria?**
The documentation of the White Box testing is a critical component of our project's success criteria. It not only demonstrates our commitment to thorough and rigorous internal testing but also ensures that all aspects of the codebase meet our quality standards. This thorough documentation will aid in any future code reviews, audits, or enhancements, and contributes to the overall reliability and integrity of our software system.

**Start date and end date:**
[Start Date: 16/11/23]
[End Date: 19/11/23]
